builder/builder.Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM debian:jessie
+
+RUN apt-get update && \
+    apt-get install -y curl debootstrap bzip2 && \
+    apt-get clean
+
+ENV WORKDIR /var/builder
+WORKDIR ${WORKDIR}
+
+ARG DOCKER_VERSION=1.11.2
+RUN curl -sSLO https://github.com/docker/docker/archive/v${DOCKER_VERSION}.tar.gz && tar -xzf v${DOCKER_VERSION}.tar.gz
```

Comment:
since you don't pass `-t` and do the `docker build` from the image outside of `mkimage.sh`, do you need the `docker` binary in the builder image?